dashboard/db/migrate/20160519000515_drop_plc_enrollment_task_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class DropPlcEnrollmentTaskAssignments < ActiveRecord::Migration
+  def change
+    drop_table :plc_enrollment_task_assignments
```

Comment:
This migration is irreversible:
```
$ rake db:rollback STEP=2
== 20160519000515 DropPlcEnrollmentTaskAssignments: reverting =================
rake aborted!
StandardError: An error has occurred, all later migrations canceled:
To avoid mistakes, drop_table is only reversible if given options or a block (can be empty).

/Users/joshlory/Work/2k14/code-dot-org/dashboard/db/migrate/20160519000515_drop_plc_enrollment_task_assignments.rb:3:in `change'

Tasks: TOP => db:rollback
(See full trace by running task with --trace)
```